tests/StarterTheme/specs/customer/my-account.js

```diff
@@ -45,5 +45,32 @@ describe('My account page', function () {
       ;
     });
 
+    it('should display credit splips link', function () {
+      return browser
+        .isExisting('.page-content a#order-slips-link')
```

Comment:
why you're using a# div# anywhere? I think it is not necessary because there should be only one unique identifier on page, right?